auditpool/run_audit.py

```diff
@@ -14,10 +14,11 @@
 TS = "ts"
 LOG_DIR = "/logs/"
 LOG_KEY = "new_val"
+DAY_STRING_RAW = asctime(gmtime(time())).split(" ")
```

Comment:
malicious compliance  for accurate time and require escaped characters 
>>> datetime.datetime.now().isoformat()
'2018-09-10T15:57:28.595586'